cfgov/v1/__init__.py

```diff
@@ -186,10 +186,10 @@ def related_metadata_tags(context, page):
     # From an ancestor, get the form ids then use the first id since the
     # filterable list on the page will probably have the first id on the page.
     id, filter_page = get_filter_data(page)
-    for tag in page.specific.tags.names():
-        tag_link = {'text': tag, 'url': ''}
+    for tag in page.specific.tags.all():
+        tag_link = {'text': tag.name, 'url': ''}
         if id is not None and filter_page is not None:
-            param = '?filter' + str(id) + '_topics=' + tag
+            param = '?filter' + str(id) + '_topics=' + tag.slug
```

Comment:
not sure what values are allowed for tag.slug, but you might want to urlencode this.

```
from urllib import urlencode
params = {"filter": id, "_topics": tag.slug
param = "?%s" % urlencode(params)
```